test/python/test_reactor.py

```diff
@@ -656,17 +655,26 @@ def test_valve_timing(self):
         valve = ct.Valve(self.r1, self.r2)
         k = 2e-5
         valve.valve_coeff = k
-        valve.set_time_function(lambda t: t>.01)
+        valve.time_function = lambda t: t > .01
 
+        delta_p = lambda: self.r1.thermo.P - self.r2.thermo.P
         mdot = lambda: valve.valve_coeff * (self.r1.thermo.P - self.r2.thermo.P)
         self.net.initialize()
-        self.assertEqual(valve.mass_flow_rate, 0.0)
+        assert valve.time_function == 0.0
+        assert valve.pressure_function == pytest.approx(delta_p())
+        assert valve.mass_flow_rate == 0.0
```

Comment:
Since we use it so often, this file already has `from pytest import approx` at the top.